packages/angular_devkit/build_webpack_nodejs/src/copy-pasted-files/read-tsconfig.ts

```diff
@@ -0,0 +1,15 @@
+// tslint:disable
```

Comment:
These files should be moved somewhere common to the projects, maybe into devkit core?